R/Seurat.R

```diff
@@ -839,7 +823,9 @@ from_Seurat <- function(
       # fetch X
       # trackstatus: class=Seurat, feature=set_X, status=done
       if (!is.null(x_mapping)) {
-        adata$X <- Matrix::t(SeuratObject::LayerData(seurat_assay, x_mapping))
+        adata$X <- to_py_matrix(
+          (SeuratObject::LayerData(seurat_assay, x_mapping))
```

Comment:
Do we need this set of brackets?

```suggestion
          SeuratObject::LayerData(seurat_assay, x_mapping)
```